src/angular/components/send/add-edit.component.ts

```diff
@@ -183,13 +189,9 @@ export class AddEditComponent implements OnInit {
             }
         }
 
-        this.disableAnonymousSend = disableSendPolicies.some(p =>
-            p.data?.disableSend === DisableSendType.DisableAnonymous);
-        this.disableSend = (this.disableAnonymousSend && this.editMode && this.send.hideEmail) ||
-            disableSendPolicies.some(p =>
-                (p.data?.disableSend ?? DisableSendType.DisableAll) === DisableSendType.DisableAll);
-
         this.hasPassword = this.send.password != null && this.send.password.trim() !== '';
+        this.disableThisSend = this.disableThisSend ||
```

Comment:
What is the purpose of this? We should be allowing a user, assuming a policy disabling anonymous email was created or added to their org after the Send's created, to edit the Send after the policy comes into effect vs. disabling everything.

The user should be able to edit the Send, disable the checkbox, but ensure it is unchecked from this perspective. So if they save it, it will save w/o the value checked.

Reply:
I think you're looking at an old commit - I changed this so that it works how you describe, unless something weird is going on with my commits

Reply:
🤦🏻